src/main/java/core/basesyntax/service/impl/FileParserServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileParserServiceImpl implements FileParserService {
```

Comment:
```suggestion
public class ParserServiceImpl implements ParserService {
```
you don't work with file here